scrapy/czlscrape/__init__.py

```diff
@@ -1,7 +1,11 @@
 import os
+import time
+import logging
 
 if 'SENTRY_DSN' in os.environ:
     import logging
     from raven.handlers.logging import SentryHandler
     from raven.conf import setup_logging
     setup_logging(SentryHandler(os.environ['SENTRY_DSN'], level=logging.WARN))
+
+logging.Formatter.converter = time.gmtime
```

Comment:
> What I don't like about local datetimes is the daylight saving differences, particularly having to cope with them.

Well, ISO-8601 preserves the offset from UTC, so it includes the daylight savings differences. And it's the job of date/time libraries to worry about conversions.

I'm actually more worried about the hackish implementation in this PR: the `gmtime` conversion and formatting the date with a hardcoded `Z` at the end. This is to work around the default behaviour of Python logging that uses timestamps in the local timezone. So maybe a compromise solution is to let the scraper log in the default timezone, and configure the production machine to UTC :)